configuration_test.go

```diff
@@ -41,7 +41,7 @@ func TestConfiguration_getICEServers(t *testing.T) {
 
 func TestConfigurationJSON(t *testing.T) {
 	j := `{
-    "iceServers": [{"URLs": ["turn:turn.example.org"],
+    "iceServers": [{"urls": ["turn:turn.example.org"],
```

Comment:
@wdouglass Is this a breaking change? 

Is it possible we can accept `URLs` and `urls` so we don't break existing users?

Reply:
I'm not sure how this test  ever worked -- if you look in iceserver.go, that field always serializes/deserializes to "urls" anyway -- this was a faulty test.

also, in the equivalent structure in "upstream" webrtc, the field is "urls"